waspc/src/Wasp/Generator/Setup.hs

```diff
@@ -10,12 +10,20 @@ import Wasp.Generator.Common (ProjectRootDir)
 import qualified Wasp.Generator.DbGenerator as DbGenerator
 import Wasp.Generator.Monad (GeneratorError (..), GeneratorWarning (..))
 import Wasp.Generator.NpmInstall (installNpmDependenciesWithInstallRecord, isNpmInstallNeeded)
+import qualified Wasp.Generator.SdkGenerator as SdkGenerator
 import qualified Wasp.Message as Msg
 
 runSetup :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> Msg.SendMessage -> IO ([GeneratorWarning], [GeneratorError])
-runSetup spec dstDir sendMessage = do
-  runNpmInstallIfNeeded spec dstDir sendMessage >>= \case
-    npmInstallResults@(_, []) -> (npmInstallResults <>) <$> setUpDatabase spec dstDir sendMessage
+runSetup spec projectRootDir sendMessage = do
+  runNpmInstallIfNeeded spec projectRootDir sendMessage >>= \case
+    npmInstallResults@(_, []) ->
+      setUpDatabase spec projectRootDir sendMessage >>= \case
+        setUpDatabaseResults@(_, []) -> do
+          -- todo(filip): Should we consider building SDK as part of code generation?
+          -- todo(filip): Avoid building on each setup if we don't need to.
```

Comment:
Todo: This takes a while and happens every time, we should definitely look into it.